charts/bulk-scan-processor/values.aat.template.yaml

```diff
@@ -5,7 +5,9 @@ java:
     DOCUMENT_MANAGEMENT_URL: "http://dm-store-aat.service.core-compute-aat.internal"
     SCAN_ENABLED: "true"
     SCAN_DELAY: "4000"
+    REPORTS_CRON: "0 0 0 1 1 ?"
```

Comment:
1st Jan at midnight?

Reply:
apparently this is a value instead of having a default assigned in java code, requested by reviewer in previous PR :|

Reply:
Is there a reason we are enabling it for PR ?